modules/site-icon/jetpack-site-icon.php

```diff
@@ -68,14 +68,21 @@ public function register_hooks(){
 		add_filter( 'display_media_states',   array( $this, 'add_media_state' ) );
 		add_action( 'admin_init',             array( $this, 'admin_init' ) );
 		add_action( 'admin_init',             array( $this, 'delete_site_icon_hook' ) );
-		add_action( 'atom_head', array( $this, 'atom_icon' ) );
-		add_action( 'rss2_head', array( $this, 'rss2_icon' ) );
 
 		add_action( 'admin_print_styles-options-general.php', array( $this, 'add_general_options_styles' ) );
 
-		// Add the favicon to the front end and backend
-		add_action( 'wp_head',           array( $this, 'site_icon_add_meta' ) );
-		add_action( 'admin_head',        array( $this, 'site_icon_add_meta' ) );
+		// Add the favicon to the front end and backend if Core's site icon not used.
+		/**
+		 * As of WP 4.3 and JP 3.6, both are outputting the same icons so no need to fire these.
+		 * This is a temporary solution until Jetpack's module primary function is deprecated.
+		 * In the future, Jetpack's can output other sizes using Core's icon.
+		 */
+		if ( ! function_exists('has_site_icon') || ! has_site_icon() ) {
```

Comment:
To avoid errors, should be 
`if ( ( function_exists( 'has_site_icon' ) && ! has_site_icon() ) || ! function_exists( 'has_site_icon' ) )`